README.md

```
@@ -214,13 +214,11 @@ func Function(
 
 
 ### Update eco-goinfra modules - How to:
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)